locale/en/about/governance.md

```diff
@@ -4,136 +4,34 @@ layout: about.hbs
 ---
 # Project Governance
 
-## Technical Steering Committee
-
-The project is jointly governed by a Technical Steering Committee (TSC)
-which is responsible for high-level guidance of the project.
-
-The TSC has final authority over this project including:
-
-* Technical direction
-* Project governance and process (including this policy)
-* Contribution policy
-* GitHub repository hosting
-* Conduct guidelines
-* Maintaining the list of additional Collaborators
-
-Initial membership invitations to the TSC were given to individuals who
-had been active contributors, and who have significant
-experience with the management of the project. Membership is
-expected to evolve over time according to the needs of the project.
+## Consensus Seeking Process
 
-For the current list of TSC members, see the project
-[README.md](https://github.com/nodejs/node/blob/master/README.md#tsc-technical-steering-committee).
+The Node.js project follows a [Consensus Seeking][] decision making model.
 
 ## Collaborators
 
-The [nodejs/node](https://github.com/nodejs/node) GitHub repository is
-maintained by the TSC and additional Collaborators who are added by the
-TSC on an ongoing basis.
+The [nodejs/node][] core GitHub repository is maintained by the Collaborators
+who are added by the TSC on an ongoing basis.
```

Comment:
Nit: Since this is the first occurrence of `TSC` in this version of the page, spell it out as `Technical Steering Committee (TSC)`.

Reply:
Ack. I reordered the paragraphs after editing.